packages/web-components/src/components/va-checkbox-group/va-checkbox-group.tsx

```diff
@@ -9,6 +9,9 @@ import {
   h,
 } from '@stencil/core';
 
+/**
+ * @vaChange The event emitted when the input value changes.
```

Comment:
Are these JSDoc decorators on the `@Component` how stencil builds up its `listeners` object/array in the generated JSON?

Reply:
These JSDoc tags get added to the `docsTags` array inside the object generated by Stencil `component-docs.json`. There is nothing in the docsTags that indicate it belongs to a specific component. An example docsTags array could look like this: 
```js
[ {  name: 'accordionItemToggled', text: 'Hey there!' } ]
```

Since there is no unique identifier, this could create issues with listeners that have the same names (for example keydown).

Reply:
We could probably improve it by adding the component name in the text for each listener in the JSDoc tag comment, and then perform some logic to remove the component name and display the rest of the text.

Reply:
These decorators get added to the `docsTags` array for each component doc generated by Stencil's `component-docs.json`, but do not extend the `listeners` with this additional information automatically.